frontend/src/reducer/workspaceList.js

```diff
@@ -58,28 +61,25 @@ function addWorkspaceSetting (setting, user, workspace, workspaceList) {
   }
   const spaceList = [
     ...workspaceList,
-    serializeRole(settings)
+    serializeUserWorkspaceConfig(settings)
   ]
   return spaceList
 }
 
 export function workspaceList (state = [], action, lang) {
   switch (action.type) {
-    case `${SET}/${ROLE_WORKSPACE_LIST}`:
-      return action.workspaceList.map(serializeRole)
-
-    case `${SET}/${WORKSPACE_LIST}`:
-      return action.workspaceList.map(serializeWorkspace)
+    case `${SET}/${USER_WORKSPACE_CONFIG_LIST}`:
```

Comment:
the const previously mentioned in a comment should be used here too

Reply:
done